packages/medusa/src/services/cart.ts

```diff
@@ -1621,13 +1621,13 @@ class CartService extends BaseService {
                   unit_price: availablePrice.calculatedPrice,
                 })
             } else {
-              await this.lineItemService_
+              return this.lineItemService_
```

Comment:
question: is the return type here falsy? Otherwise, I'd expect the filter operation below to have issues with removing the item?

Reply:
it is a mistake I wanted to just return; instead of return null